### DISCUSSION TITLE
Start up pilot position

### DISCUSSION THREAD

**Discussion by Harpz1991** [flybywiresim/aircraft#7032] [Ideas]
Is it possible to have an selectable option in the efb for start up camera position.  Instead of having to click copilot to move to the right seat every time you load the aircraft

**Comment by ImenesFBW** [comment]
Good suggestion! In the meantime, you can use the "VFR Pilot View Save" command, default binding CTRL+F10. If you move to copilot position and press that binding the camera will default to that position. You can reset it back by pressing SHIFT+F10
